plugins/modules/system/filesystem.py

```diff
@@ -30,25 +32,23 @@
     default: present
     version_added: 1.3.0
   fstype:
-    choices: [ btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap ]
+    choices: [ btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap, ufs ]
     description:
       - Filesystem type to be created. This option is required with
         C(state=present) (or if I(state) is omitted).
-      - reiserfs support was added in 2.2.
-      - lvm support was added in 2.5.
-      - since 2.5, I(dev) can be an image file.
-      - vfat support was added in 2.5
-      - ocfs2 support was added in 2.6
-      - f2fs support was added in 2.7
-      - swap support was added in 2.8
+      - ufs support has been added in 3.4.0 of community.general.
     type: str
     aliases: [type]
   dev:
     description:
-      - Target path to block device or regular file.
-      - On systems not using block devices but character devices instead (as
-        FreeBSD), this module only works when applying to regular files, aka
-        disk images.
+      - Target path to block device (Linux) or character device (FreeBSD) or
+        regular file (both).
+      - When setting Linux-specific filesystem types on FreeBSD, this module
+        only works when applying to regular files, aka disk images.
+      - Currently C(lvm) (Linux-only) and C(ufs) (FreeBSD-only) don't support
+        a regular file as their target I(dev).
+      - Support for character devices on FreeBSD has been added in 3.4.0 of
+        comunity.general.
```

Comment:
Writing versions this way makes it easier to search for them with grep:
```suggestion
      - Support for character devices on FreeBSD has been added in community.general 3.4.0.
```

Reply:
that's done